projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.scss

```diff
@@ -0,0 +1,112 @@
+@import "projects/commudle-admin/src/assets/styles/variables";
+
+.home-events-card {
+  width: 256px;
+  border: 0.5px solid $silver-sand;
+  box-sizing: border-box;
+  position: relative;
+
+  .home-events-attending {
+    position: absolute;
+    height: 20px;
+    top: 8px;
+    right: 0;
+    border-radius: $border-5 0 0 $border-5;
+    background-color: $neon-blue;
+    color: $white;
+    padding: $space-4;
+    font-size: 11px;
+    font-family: $primary-font-family;
+    font-weight: 500;
+    line-height: 13px;
+    z-index: 2;
+
+    nb-icon {
+      font-size: 11px;
+    }
+  }
+
+  .home-events-image {
+    height: 156px;
+    overflow: hidden;
+
+    img {
+      width: 100%;
+      height: inherit;
+      object-fit: cover;
+      object-position: left top;
+    }
+
+    app-skeleton-text {
+      &::ng-deep {
+        .row {
+          height: 156px;
+        }
+      }
+    }
+  }
+
+  .home-events-date {
+    position: absolute;
+    height: 56px;
+    width: 56px;
+    top: 120px;
+    right: 12px;
+    border-radius: $border-8;
+    background-color: $neon-blue;
+    color: $white;
+    padding: $space-8 $space-12;
+    font-size: 16px;
+    font-family: $primary-font-family;
+    font-weight: 500;
```

Comment:
Let's get standard font-weights from Apoorva, she said she's added them to figma, can you please figure this one out

Reply:
Done!